DigraphyApi/Mapper/MappingProfiles.cs

```diff
@@ -0,0 +1,20 @@
+using AutoMapper;
+using DigraphyApi.Dto;
+using DigraphyApi.Models;
+
+namespace DigraphyApi.Mapper;
+
+// can't rememeber if we need this or not
+public class MappingProfiles : Profile
+{
+    public MappingProfiles()
+    {
+        // Todo mapping
+        CreateMap<Todo, TodoDto>();
+        CreateMap<TodoDto, Todo>()
+            .ForMember(dest => dest.Id, opt => opt.Ignore());
+        CreateMap<Todo, CreateTodoDto>();
```

Comment:
Is this necessary? We shouldn't ever be returning a `CreateTodoDto`, we'll always be mapping from a `Todo` to a `TodoDto`.